Jenkinsfile

```diff
@@ -18,7 +21,13 @@ pipeline {
     }
     stage('Security audit') { 
```

Comment:
```
stage('Security audit') {
  when {
    expression { params.AUDIT == 'true' }
  }
  steps {
...
  }
}
```

Reply:
Nice I hadn't seen this, very clean solution 👍